kbase-extension/static/kbase/js/common/cellComponents/tabs/infoTab.js

```diff
@@ -1,47 +1,78 @@
-define(['common/format'], (format) => {
+define(['common/format', 'common/html', 'util/string'], (format, html, string) => {
```

Comment:
converted this over to use the kbase `html` module -- why would you want to use DOM manipulation when there is a very fine kbase alternative?! It makes it a bit easier to mess with classes, see the output structure, etc.